packages/deck/backend/sirius-components-deck/src/main/java/org/eclipse/sirius/components/deck/description/LaneDescription.java

```diff
@@ -29,7 +29,17 @@
 @PublicApi
 public record LaneDescription(String id, Function<VariableManager, String> targetObjectKindProvider, Function<VariableManager, String> targetObjectLabelProvider,
         Function<VariableManager, String> targetObjectIdProvider, Function<VariableManager, List<Object>> semanticElementsProvider, Function<VariableManager, String> titleProvider,
-        Function<VariableManager, String> labelProvider, List<CardDescription> cardDescriptions, Consumer<VariableManager> editLaneProvider, Consumer<VariableManager> createCardProvider) {
+        Function<VariableManager, String> labelProvider, List<CardDescription> cardDescriptions, Consumer<VariableManager> editLaneProvider, Consumer<VariableManager> createCardProvider, Consumer<VariableManager> dropCardProvider) {
+
+
+    public static final String NEW_LABEL = "newLabel";
+    public static final String TITLE = "title";
+    public static final String DESCRIPTION = "description";
+    public static final String OLD_LANE = "oldLane";
```

Comment:
The first three variables are never used

Reply:
done